t/api/14-plugin_obs_rsync_async.t

```diff
@@ -19,10 +19,10 @@ $t->app($app);
 
 sub start_gru {
     start sub {
-        note('starting gru');
-        $0 = 'openqa-gru';
-        $ENV{MOJO_MODE} = 'test';
-        Mojolicious::Commands->start_app('OpenQA::WebAPI', 'gru', 'run', '-m', 'test');
+        note('starting gru');    # uncoverable statement
+        $0 = 'openqa-gru';    # uncoverable statement
+        $ENV{MOJO_MODE} = 'test';    # uncoverable statement
+        Mojolicious::Commands->start_app('OpenQA::WebAPI', 'gru', 'run', '-m', 'test');    # uncoverable statement
```

Comment:
Where is this used? I see `my $gru = start_gru();` below so I would think this is always covered/executed

Reply:
Ah. I guess that makes sense. Maybe add a comment at the top to indicate that.

Reply:
This is only one of many of such situations. I would not add an extra comment about it.

Reply:
Added the comment nevertheless. At this point sprinkling comments doesn't hurt anymore anyway.